tests-system/lobster-report/lobsterreportasserter.py

```diff
@@ -0,0 +1,14 @@
+from ..asserter import Asserter
+
+
+# Setting this flag will tell unittest not to print tracebacks from this frame.
+# This way our custom assertions will show the interesting line number from the caller
+# frame, and not from this boring file.
+__unittest = True
+
+
+class LobsterReportAsserter(Asserter):
+    def assertStdOutNumAndFile(self, num_items: int, out_file: str):
+        self.assertStdOutText(
+            f""
```

Comment:
Please add an actual std out message.
```suggestion
            f"lobster-json: wrote {num_items} items to {out_file}\n"
```

Reply:
But I think `lobster-report` doesn't print any stdout messages in success scenario. So please ignore the above change.